src/components/Bomberman.tsx

```diff
@@ -0,0 +1,95 @@
+import React = require('react');
+import * as BombermanMap from './BombermanMap';
+
+interface GameState {
+  map: string[];
+  width: number;
+  higth: number;
+}
+
+export class BombermanGame extends React.Component<{}, GameState> {
+  constructor(props: {}) {
+    super(props);
+    const initMap = new Array(15 * 15).fill('empty');
+    initMap[0] = 'p1';
+    this.state = {
+      higth: 15,
+      map: initMap,
+      width: 15,
+    };
+  }
+
+  isAccesible(location: number, targetLocation: number): boolean {
+    return !(
+      targetLocation < 0 ||
+      targetLocation > this.state.width * this.state.higth ||
+      (targetLocation % this.state.width === 0 &&
+        location % this.state.width === this.state.width - 1) ||
+      (targetLocation % this.state.width === this.state.width - 1 &&
+        location % this.state.width === 0)
+    );
+  }
+
+  move(direction: string, player: string) {

```

Comment:
`direction: 'up' | 'down' | 'left' | 'right'` would get Typescript to ensure you only pass valid direction values. Probably worth declaring `type Direction: 'up' | 'down' | 'left' | 'right'` somewhere to save typing it each time.